cmd/internal/apidump/apidump.go

```diff
@@ -80,6 +84,17 @@ var Cmd = &cobra.Command{
 			outFlag.AkitaURI = &uri
 		}
 
+		// If --collection was given, set rest.domain and environment config (if given)
+		if postmanCollectionID != "" {
+			if postmanEnvironment != "" {
+				env := strings.ToUpper(postmanEnvironment)
+
+				cfg.WritePostmanEnvironment("default", env)
```

Comment:
I don't understand why you are writing to the config file here.  `apidump` should not alter any configuration files, only `login` should do so.

Reply:
Wasn't aware that config files should only be updated via `login` command, removed this from both `apidump` and `kube inject`